package.json

```diff
@@ -148,6 +149,7 @@
   "nextBundleAnalysis": {
     "budget": null,
     "budgetPercentIncreaseRed": 20,
-    "showDetails": true
+    "showDetails": true,
+    "minimumChangeThreshold": 1000
```

Comment:
So many bytes!

Reply:
How about we go to 300 and see how that feels? Anything under that and we'll be reporting pages as changed when it's really just Next adding small bits of metadata to pages (for example, [this PR](https://github.com/hashicorp/dev-portal/pull/115)).

Reply:
Yeah, I mean I'm totally fine with 1k here too. We have the risk of death by a thousand cuts in that a bunch of tiny changes over time might not be visible until we're in a bad spot, but the noise reduction is definitely nice, and we should really be storing the analysis data and graphing it over time to track trends and such too.